topology/connection_creator.h

```diff
@@ -208,10 +216,28 @@ ConnectionCreator::connect_( index s,
       // TODO implement in terms of nest-api
       kernel().connection_manager.connect(
         s, target, target_thread, syn, d, w );
+      if ( make_symmetric_ )
+      {
+        make_symmetric_connection_( s, target, target_thread, syn, d, w );
+      }
     }
   }
 }
 
+inline void
+ConnectionCreator::make_symmetric_connection_( const index& s,
+  const Node* target,
+  const thread& target_thread,
+  const index& syn,
+  const double& d,
+  const double& w )
+{
+  const index s_thread = kernel().vp_manager.get_thread_id();
+  Node* const s_node = kernel().node_manager.get_node( s, s_thread );
+  kernel().connection_manager.connect(
+    target->get_gid(), s_node, s_thread, syn, d, w );
+}
+
```

Comment:
Is this also working for parallel simulations? I am asking since connection rules as `fixed_total_number` etc. cannot be implemented with symmetric connections in their current version (which is why they are not allowed with gap junctions)